cmd/up/activate.go

```diff
@@ -194,7 +194,7 @@ func (up *upContext) activate() error {
 			if isWatchesConfigurationTooLow(watches) {
 				folder := config.GetNamespaceHome(up.Dev.Namespace)
 				if utils.GetWarningState(folder, ".remotewatcher") == "" {
-					oktetoLog.Yellow("The value of /proc/sys/fs/inotify/max_user_watches in your cluster nodes is too low.")
+					oktetoLog.Yellow("The value of /proc/sys/fs/inotify/max_user_watches in your context nodes is too low.")
```

Comment:
```suggestion
					oktetoLog.Yellow("The value of /proc/sys/fs/inotify/max_user_watches in your cluster nodes is too low.")
```
I thing in this case is clearer talk about cluster